src/nyc_trees/apps/quiz/models.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+
+
+class Quiz(object):
```

Comment:
making a separate app for quizzes is probably a good thing, and using a class to model a quiz is also probably a good thing. However, I think having a `models.py` that lives in a django `app` but does not contain django models is a recipe for confusion, given that `models.py` is such a ubiqutious convention in django.

In a PR for a similar feature, I made a `training` python package inside the `home` app, but it probably deserves its own app. I also ran into this `models.py` problem, and I chose `types.py`, but I can't say I feel great about that choice of name.